src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family should include a generic font family as a fallback. In this case, 'sans-serif' is already specified, but it's a good practice to also include 'Arial' or another similar web-safe font before the generic family.